trunk/source/CodeCheck/src/de/uni_freiburg/informatik/ultimate/plugins/generator/codecheck/preferences/CodeCheckPreferenceInitializer.java

```diff
@@ -121,7 +121,7 @@ public enum RedirectionStrategy {
 	public static final boolean DEF_USESEPARATETRACECHECKSOLVER = true;
 	public static final SolverMode DEF_CHOOSESEPARATETRACECHECKSOLVER = SolverMode.Internal_SMTInterpol;
 	public static final String DEF_SEPARATETRACECHECKSOLVERCOMMAND = "";
-	public static final String DEF_SEPARATETRACECHECKSOLVERTHEORY = "QF_AUFLIA";
+	public static final String DEF_SEPARATETRACECHECKSOLVERTHEORY = "ALL";

```

Comment:
Is it really intended to change the default setting here? If so, are there any settings files that also need to be adapted?

Reply:
In all the relevant settings (e.g. for SV-COMP) `ALL` was already used. The most simple solution to fix the failing test would be of course to just adapt the settings files that is responsible for the tests. However, to be consistent with Automizer (see `RcfgPreferenceInitializer`), I adapted the default setting for Kojak instead, but maybe this is nothing that should be done in this PR.